### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_About_Zoom: At 200% zoom "About" link is not visible in the page.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#49702]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopWeb; #BM_.NETCore_Web_Jul23; #.NET Core; #WCAG1.4.4; #Win11; #Zoom; #ChromiumEdge; #A11ypreview;

### Environment Details:
App name: .NET Core
URL: http://localhost:5268/

Chromium Edge: Version 116.0.1938.29 (Official build) dev (64-bit)

### Repro Steps:

1. Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
2. Move to the created directory using command "cd foldername".
3. Run command "dotnet new blazorwasm" and hit ENTER key.
4. Run command "dotnet run".
5. Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.
6. Zoom the screen to 200% (Display Resolution: 1280 * 768).
7. TAB to "About" link and observe that "About" link is not visible in the page.

### Actual Result:
At 200% zoom "About" link is not visible in the page.

### Expected Result:
At 200% zoom "About" link should be visible clearly in the page.

### User Impact:
Low vision users will not be able to see the link and will not get information about.

### Attachment:

https://github.com/dotnet/aspnetcore/assets/90901519/feb2950f-c192-4c4c-ac69-ec8fc8fbf70a

**Comment by merriemcgaw** [comment]
@mkArtakMSFT @SteveSandersonMS - this is another one required for RC2. Can you have someone take a look?

**Comment by mkArtakMSFT** [comment]
#Closed;

**Comment by merriemcgaw** [comment]
@PoojaNamde this is another issue to confirm.

**Comment by PoojaNamde** [comment]
#Reopened; #Regressed:09-15-23;

@merriemcgaw Verified this issue in the environment http://localhost:5227/, but issue is still able to repro. hence reactivating the bug.

![image](https://github.com/dotnet/aspnetcore/assets/90901519/944a8747-98cd-464c-a94c-bd68d717280b)

**Comment by kupatkar99** [comment]
#Closed; #Regressed:10-04-23;
Verified the issue in latest environment
Browser details: Microsoft Edge
Version 119.0.2132.0 (Official build) dev (64-bit)
Issue is fixed. Hence Closing the bug Please find added attachment for reference.
![49702_fixed](https://github.com/dotnet/aspnetcore/assets/98530069/88a1b8fb-0c60-4e30-9e0c-f91ec40dc3b1)
